pkg/term/term_windows.go

```diff
@@ -86,10 +140,17 @@ func SaveState(fd uintptr) (*State, error) {
 // -- See http://msdn.microsoft.com/en-us/library/windows/desktop/ms683462(v=vs.85).aspx
 func DisableEcho(fd uintptr, state *State) error {
 	mode := state.mode
-	mode &^= winconsole.ENABLE_ECHO_INPUT
-	mode |= winconsole.ENABLE_PROCESSED_INPUT | winconsole.ENABLE_LINE_INPUT
-	// TODO(azlinux): Core code registers a goroutine to catch os.Interrupt and reset the terminal state.
-	return winconsole.SetConsoleMode(fd, mode)
+	mode &^= windows.ENABLE_ECHO_INPUT
+	mode |= windows.ENABLE_PROCESSED_INPUT | windows.ENABLE_LINE_INPUT
+
+	err := windows.SetConsoleMode(fd, mode)
+	if err != nil {
```

Comment:
@RickWieman I'd seen both. I must have missed that comment in Effective Go. I can change this.